### PR TITLE
Fixes for python 3.7 and 3.8

### PR DESCRIPTION
**Pull request by MasloMaslane**
There were some problems with types and `importlib.resources` in python 3.7 and 3.8.

### CODE REVIEW

**Comment by tonowak** [comment]
Did you add any tests that checks whether the problem is indeed solved?

**Comment by MasloMaslane** [comment]
I ran them locally on both 3.7 and 3.8. Do you think that we should run all tests on 3.7 and 3.11? This would make the tests even longer

**Comment by tonowak** [comment]
If we can run in parallel, why not?

**Comment by tonowak** [comment]
Is it better to have users for which the code doesn't work, or wait a bit with merging the PRs? :)

**Comment by MasloMaslane** [comment]
Our local runner can't run more than one job at once, but I'll try adding more and see if they still work

**Comment by tonowak** [comment]
Probably running on the oldest and latest Python version which we support will be enough.

**Comment by MasloMaslane** [comment]
I added a new runner and now the tests are on 3.7 and 3.11

File: src/sinol_make/util.py [hunk 1]
```diff
@@ -204,6 +210,12 @@ def check_version():
     Function that asynchronously checks for new version of sinol-make.
     Writes the newest version to data/version file.
     """
+    python_version = sys.version_info
+    if python_version.minor <= 8:
+        import importlib_resources as importlib
+    else:
+        import importlib.resources as importlib
+
```
Comment (tonowak) [inline src/sinol_make/util.py:218]: code duplication?

Comment (MasloMaslane) [reply src/sinol_make/util.py]: I added a function for importing the library

**Review by tonowak — APPROVED** [review]

**Pull request merged**